src/OmniSharp.Roslyn.CSharp/project.json

```diff
@@ -6,9 +6,9 @@
   "dependencies": {
     "OmniSharp.Abstractions": "1.0.0",
     "OmniSharp.Roslyn": "1.0.0",
-    "Microsoft.CodeAnalysis.CSharp": "1.3.0",
-    "Microsoft.CodeAnalysis.CSharp.Features": "1.3.0",
-    "Microsoft.CodeAnalysis.CSharp.Workspaces": "1.3.0"
+    "Microsoft.CodeAnalysis.CSharp": "2.0.0-beta3",
+    "Microsoft.CodeAnalysis.CSharp.Features": "2.0.0-beta3",
```

Comment:
I just took a look at the following files in Roslyn, that OmniSharp reflects into. It looks like they're OK.

* [CSharpMetadataAsSourceService.cs](https://github.com/dotnet/roslyn/blob/master/src/Features/CSharp/Portable/MetadataAsSource/CSharpMetadataAsSourceService.cs) - Last updated 9 months ago
* [MetadataAsSourceHelpers.cs](https://github.com/dotnet/roslyn/blob/master/src/Features/Core/Portable/MetadataAsSource/MetadataAsSourceHelpers.cs) - Recent changes were to update resource string IDs. Before that it was 9 months.
* [SymbolKey.cs](https://github.com/dotnet/roslyn/blob/master/src/Workspaces/Core/Portable/SymbolKey/SymbolKey.cs) - This has more significant changes but it looks like the API we use [here](https://github.com/OmniSharp/omnisharp-roslyn/blob/dev/src/OmniSharp.Roslyn/MetadataHelper.cs#L81-L87) is essentially [the same](https://github.com/dotnet/roslyn/blob/master/src/Workspaces/Core/Portable/SymbolKey/SymbolKey.cs#L123-L126).